recipes/coremltools/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+mkdir build
+cd build
+
+cmake \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 \
```

Comment:
If this is actually needed, see: https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks

(I don't think it will do anything in staged recipes but it will in the feedstock)